standards/bulk.md

```diff
@@ -0,0 +1,134 @@
+- Bulk operations **MUST** be implemented as a PATCH request against a collection resource.
+- Bulk operations **MUST** return a `200 (OK)` response code if all operations were received and a result is available for each operation, regardless of success. Bulk operations **MUST NOT** use status code `207 (Multi-Status)` response code as this incurs other implications for the response schema in relationship to Web DAV.
+- Bulk operations **MUST** accept a constrained number of operations in the request body that is indicated in the documentation of the endpoint. By default this value **MAY** be 100 operations, but should be adjusting according to the needs of the endpoint and the entity-size. Requests beyond the limit **MUST** return a `400` response code and standard [error format body](errors.md) similar to:
+```
+// RESPONSE
+HTTP/1.1 400
+Content-Type: application/problem+json
+{
+    "title": "Invalid Data",
+    "status": 400,
+    "detail": "Operations collection may only contain a maximum of '100' actions per request.",
+    "instance": "/articles",
+    "requestId": "b6d9a290-9f20-465b-bcd3-4a5166eeb3d7"
+}
+```
+
+- Bulk operations **MUST** include a request body schema following: 
+```json
+{
+    "operations": [
+        {
+            "action": "CREATE" | "UPDATE" | "CREATE_UPDATE" | "DELETE",     // indicate intent of operation
+            "ifMatch": "string" | null,                                     // if-match is an optional ETag that can be passed for optimistic concurrency
+            "entity": {
+                ... MATCHING ENTITY...                                      // must match entity schema resource from the collection
+            }                                                               // schema on entity is static and not dynamic
+        },
+    ]
+}
+```
+
+- Bulk operations **MUST** include a response body schema following:
+```json
+{
+    "status": "SUCCEEDED" | "FAILED" | "PARTIAL",                           // overall status of bulk operation, partial indicating 
+    "operations": [                                                         // there are some operations that failed and succeeded
+        {
+            "ordinal": 0,                                                   // position in original ordered operation collection
+            "action": "CREATE" | "UPDATE" | "CREATE_UPDATE" | "DELETE",     // repeat action type
+            "id": "string" | null,                                          // the associated id of the entity, if available
+            "ref": "sps-ref" | null,                                        // the associated sps-ref URN entity, if applicable
+            "status": "SUCCEEDED" | "FAILED",                               // status of individual operation
+            "detail": "string",                                             // indicates details on the operation result, such as a error message.
```

Comment:
For the `FETCH` mentioned above, would we maybe want to go for `data`, as well?  Something that might be api dependent.  In the (possible) Item Bulk Lookup it would generically be an `object`.  Maybe we'd want a `schema` ref in those cases 🤔

Reply:
FETCH will be out of scope for this purpose.